bentoml/_internal/runner/runnable.py

```diff
@@ -24,15 +23,31 @@
 RUNNABLE_METHOD_MARK: str = "_bentoml_runnable_method"
 
 
-class Runnable(ABC):
-    SUPPORT_NVIDIA_GPU: bool
+class Runnable:
+    SUPPORTED_RESOURCES: set[str]
     SUPPORT_CPU_MULTI_THREADING: bool
 
-    methods: dict[str, RunnableMethod[t.Any, t.Any, t.Any]] | None = None
+    bentoml_runnable_methods__: dict[
```

Comment:
can we make it a special attributes
```suggestion
    __bentoml_runnable_methods__: dict[
```

Reply:
seems like a good idea to have a special `__` that is unique to runners and more pythonic

Reply:
I don't think this name needs to be internal to `BentoML`. It's something that can be public, which is why I originally named it `methods`.

Probably we don't need the underscores if we named it `bentoml_`... if you don't think that's nice.

Reply:
it just seems nonsymmetrical. but again this is more of a codestyle thing.